promscale: Add doc about usage of jaeger grpc storage (#1536)
timescale/docs: promscale/send-data/jaeger.md

---

# Send Jaeger traces to Promscale

> Promscale natively supports both Jaeger gRPC based remote storage(Promscale >= 0.14.0) and
> OpenTelemetry Protocol (OTLP) for traces.

Reviewer: ```suggestion
On Promscale&nbsp;0.14.0 and above, you can natively use both Jaeger 
gRPC based remote storage and OpenTelemetry Protocol (OTLP) for traces.
```

Author: Thanks @Loquacity for your suggestions.

"OpenTelemetry Protocol (OTLP) for traces" has been supported since 0.11.0. Hence the above suggested statement sounds incorrect.

Reviewer: ```suggestion
On Promscale&nbsp;0.14.0 and above, you can natively use Jaeger gRPC 
based remote storage. On Promscale&nbsp;0.11.0 and above, you can natively 
use OpenTelemetry Protocol (OTLP) for traces.
```

?

> To ingest Jaeger traces to Promscale, you can use either of the following,
>  - Use the Jaeger Collector, it ingests Jaeger spans directly to configured remote storage and 
> this avoid using any non Jaeger components like OpenTelemetry collector.
>  - Use the OpenTelemetry Collector, it converts Jaeger traces to OpenTelemetry traces.

Reviewer: ```suggestion
To ingest Jaeger traces to Promscale, you can either:

* Use the Jaeger Collector to ingest Jaeger spans directly to configured 
   remote storage. This avoids using any non-Jaeger components, including 
   the OpenTelemetry collector.
* Use the OpenTelemetry Collector to convert Jaeger traces to OpenTelemetry traces.
```

# Send data using the Jaeger Collector
You can configure the Jaeger Collector to store the traces using Promscale's native 
implementation of [Jaeger gRPC storage specification][jaeger-grpc-storage].

Here's an example configuration to enable the Jaeger collector to forward traces

> to Promscale.

Reviewer: ```suggestion
to Promscale:
```

```sh
docker run \
  -e SPAN_STORAGE_TYPE=grpc-plugin \
  -e GRPC_STORAGE_SERVER="<PROMSCALE_HOST>:<gRPC_PORT>" \
  jaegertracing/jaeger-collector:1.37.0
```

Where: 
* `<PROMSCALE_HOST>`: hostname of Promscale.
* `<gRPC_PORT>`: gRPC port of Promscale. The default port is 9202.

If you are running the Jaeger Collector and the Promscale Connector on a
Kubernetes cluster, the endpoint parameter is similar to `endpoint:
"promscale-connector.default.svc.cluster.local:<PORT>"`

> Note: We recommend to use this option unless otherwise you need additional capabilities
> offered by [OpenTelemetry collector][otelcol-config].

Reviewer: ```suggestion
<highlight type="note">
This is the preferred option, unless you need the additional capabilities
of the [OpenTelemetry collector][otelcol-config].
</highlight>
```
